answerking_app/tests/test_integration/IntegrationTestBaseClass.py

```diff
@@ -53,12 +72,15 @@ def seedDictFixture(self, fixture_type, dict_data):
             Category.objects.create(**dict_data)
         elif fixture_type == "tags":
             Tag.objects.create(**dict_data)
+        elif fixture_type == "orders":
+            order_id = self.create_order_lineItems(dict_data)
```

Comment:
does create_order_lineItems return a value? I can't see a `return` in this function?